resource/supported.go

```diff
@@ -563,6 +572,14 @@ func (a *AWS) ebsVolumes() (interface{}, error) {
 	return output.Volumes, nil
 }
 
+func (a *AWS) ecsClusters() (interface{}, error) {
+	output, err := a.ListClusters(&ecs.ListClustersInput{})
+	if err != nil {
+		return nil, err
+	}
+	return output, nil
```

Comment:
Here you need to return `output.ClusterArns` :)

Reply:
I decided to try with `ListClusters` because `DescribeClusters` [needs an input Cluster to describe](https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#DescribeClustersInput).
Also `DeleteCluster` need only the Cluster's ARN, which can be obtained by the `ListClusters`.
Point taken on the `output` var, needed to be `output.ClusterArns`. But still get errors I'm trying to figure out.

Reply:
I would do a `ListClusters` and then use a `DescribeClustern` an all the returned ARNs. The advantage is then that we have all the cluster attributes to also let `awsweeper` filter on tags, for example (https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#Cluster).